src/library/entities/Library.java

```diff
@@ -17,65 +17,67 @@
-    private static final double FiNe_PeR_DaY = 1.0;
-    private static final double maxFinesOwed = 1.0;
-    private static final double damageFee = 2.0;
-    
-    private static Library SeLf;
-    private long NeXt_ItEm_Id;
-    private long NeXt_PaTrOn_Id;
-    private long NeXt_lOaN_Id;
-    private Date CuRrEnT_DaTe;
-    
-    private Map<Long, Item> CaTaLoG;
-    private Map<Long, Patron> PaTrOnS;
-    private Map<Long, Loan> LoAnS;
-    private Map<Long, Loan> CuRrEnT_LoAnS;
-    private Map<Long, Item> DaMaGeD_ItEmS;
+    private static final String LIBRARY_FILE = "library.obj";
+    private static final int LOAN_LIMIT = 2;
+    private static final int LOAN_PERIOD = 2;
+    private static final double FINE_PER_DAY = 1.0;
+    private static final double MAX_FINES_OWED = 1.0;
+    private static final double DAMAGE_FEE = 2.0;
+    
+    private static Library self;
+    private long nextItemId;
+    private long nextPatronId;
+    private long nextLoanId;
+    private Date currentDate;
+    
+    private Map<Long, Item> catalog;
+    private Map<Long, Patron> patrons;
+    private Map<Long, Loan> loans;
+    private Map<Long, Loan> currentLoans;
+    private Map<Long, Item> damagedItems;
     
 
     private Library() {
-        CaTaLoG = new HashMap<>();
-        PaTrOnS = new HashMap<>();
-        LoAnS = new HashMap<>();
-        CuRrEnT_LoAnS = new HashMap<>();
-        DaMaGeD_ItEmS = new HashMap<>();
-        NeXt_ItEm_Id = 1;
-        NeXt_PaTrOn_Id = 1;        
-        NeXt_lOaN_Id = 1;        
+        catalog = new HashMap<>();
+        patrons = new HashMap<>();
+        loans = new HashMap<>();
+        currentLoans = new HashMap<>();
+        damagedItems = new HashMap<>();
+        nextItemId = 1;
+        nextPatronId = 1;
+        nextLoanId = 1;
     }
 
     
-    public static synchronized Library GeTiNsTaNcE() {        
-        if (SeLf == null) {
-            Path PATH = Paths.get(lIbRaRyFiLe);            
+    public static synchronized Library getInstance() {
+        if (self == null) {
+            Path PATH = Paths.get(LIBRARY_FILE);
             if (Files.exists(PATH)) {    
-                try (ObjectInputStream LiBrArY_FiLe = new ObjectInputStream(new FileInputStream(lIbRaRyFiLe));) {
+                try (ObjectInputStream libraryFile = new ObjectInputStream(new FileInputStream(LIBRARY_FILE));) {
                 
-                    SeLf = (Library) LiBrArY_FiLe.readObject();
-                    Calendar.GeTiNsTaNcE().sEtDaTe(SeLf.CuRrEnT_DaTe);
-                    LiBrArY_FiLe.close();
+                    self = (Library) libraryFile.readObject();
+                    Calendar.GeTiNsTaNcE().sEtDaTe(self.currentDate);
+                    libraryFile.close();
                 }
                 catch (Exception e) {
                     throw new RuntimeException(e);
                 }
             }
-            else SeLf = new Library();
+            else {

```

Comment:
Need to conform to bracket style. Should be:

```
if (...) {
  ...
} else {
  ...
}
```

Reply:
That is not the way it is in the CSG 8 -We agreed before on 
using the first option on Discord.